apps/codeforafrica/src/payload/collections/Partners.js

```diff
@@ -50,7 +50,7 @@ const Partners = {
       name: "description",
       required: true,
       localized: true,
-      editor: slateEditor(),
+      editor: slateEditor({}),
```

Comment:
Very bad API design from Payload's part!

Reply:
Yeah, its like we are always expected to pass some configs which is far from the truth for most simple use cases